### DISCUSSION TITLE
提交时，input值如何带上prefix

### DISCUSSION THREAD

**Discussion by xiaozhahu** [alibaba/formily#2566] [Q&A 2.x] [answered]
<SchemaField.Void
  x-component="ArrayTable.Column"
  x-component-props={{ title: 'Uri'}}
>
    <SchemaField.String
      name="uri"
      x-component="Input"
      x-decorator="FormItem"
      x-component-props={{ prefix: '/user'}}
    />
</SchemaField.Void>

提交时，如何将/user拼接到value中
尝试使用onFieldSubmitStart这个hook，并没有触发
用onFormSubmitStart这个处理起来貌似有些麻烦

**Comment by janryWang** [comment, marked as answer]
prefix是组件的ui属性，你这种场景是很明显的定制组件场景，写个自定义组件
